.github/workflows/ci-cd.yml

```
@@ -379,6 +379,10 @@ jobs:
         - spellcheck-docs
         xfail:
         - false
+        include:
+        - python-version: 3.14
+          runner-vm-os: ubuntu-latest
+          toxenv:pre-commit
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)